Enterprise-CMCS/macpro-mdct-seds
# MDCT-2179: remove auth job codes table

### Description
This table is empty and is unused by any code logic

Stack successfully updates and removes table without issues
![Screenshot 2023-07-28 at 10 34 05 AM](https://github.com/Enterprise-CMCS/macpro-mdct-seds/assets/57802560/c557b2ed-2b1c-4c5d-88c2-1cb6bdcdec4b)


### Related ticket(s)
MDCT-2179

---
### How to test
See deploy still succeeds
See tests still pass
Unfortunately we can't test deployed branches very well :/ 

### Important updates
You can remove the following two from your .env

```
AUTH_JOB_CODES_TABLE_NAME
AUTH_JOB_CODES_TABLE_ARN
```
